recipes/rpaframework/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "rpaframework" %}
+{% set version = "7.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rpaframework-{{ version }}.tar.gz
+  sha256: b93663b620e70550f7eb92380ceba0612dc99e500cf4efb3d8e004ecd388bcf8
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  entry_points:
+    - rpa-google-oauth = RPA.scripts.google_authenticate:start
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry
+  run:
+    - cryptography >=2.9.2,<3.0.0
+    - dataclasses >=0.7,<0.8  # [py>=36 and py<37]
+    - docutils
+    - exchangelib >=3.1.1,<4.0.0
+    - fpdf >=1.7.2,<2.0.0
+    - python-graphviz >=0.13.2,<0.14.0
+    - jsonpath-ng >=1.5.2,<2.0.0
+    - python-mss >=6.1.0,<7.0.0
+    - netsuitesdk >=1.1.0,<2.0.0
+    - notifiers >=1.2.1,<2.0.0
+    - openpyxl >=3.0.3,<4.0.0
+    - pdfminer.six ==20201018
+    - pillow >=8.0.1,<9.0.0
+    - psutil >=5.7.0,<6.0.0  # [win]
+    - pynput-robocorp-fork >=2.0.0,<3.0.0
+    - pypdf2 >=1.26.0,<2.0.0
+    - pyperclip >=1.8.0,<2.0.0
+    - python
+    - python-xlib >=0.17  # [linux]
+    - pywin32 >=227,<228  # [py<37 and win or py>37 and py<38 and win or py>38 and win]
```

Comment:
can the condition be `[py>=37 and win]`?

Reply:
That would drop the dependency for python versions <3.7, right? But that is actually not a problem for us so we can try that one.

Reply:
Maybe it could even be `[py<=40 and win]`?